src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
 // write code here
+function main() {
+  const doc = document.documentElement;
+  const list = doc.querySelector('ul');
+
+  const children = Array.from(list.children);
+
+  children.sort(sortFunc);
+
+  for (const obj of children) {
+    list.appendChild(obj);
+  }
+}
+
+function sortFunc(a, b) {
+  const aSal = formatSalary(a.dataset.salary);
+  const bSal = formatSalary(b.dataset.salary);
+
+  return bSal - aSal;
+}
```

Comment:
The `sortFunc` function correctly sorts the list items by salary in descending order. This part of the task is implemented correctly.